dev/PushNotifications/PushNotificationUtility.h

```diff
@@ -127,9 +130,30 @@ namespace winrt::Microsoft::Windows::PushNotifications::Helpers
     }
     CATCH_RETURN()
 
+    inline void ClearRoActivateInstanceCache()
+    {
+        // Activating a nonexistent class clears the cache enables the proxyStub to be found for elevated scenarios
+        auto activatableClass{ wil::make_unique_string_nothrow<wil::unique_hstring>(L"FakeClass.ClassName") };
+        winrt::com_ptr<::IInspectable> ptr{};
+
+        RoActivateInstance(activatableClass.get(), ptr.put());
+    }
+
     inline wil::com_ptr<INotificationsLongRunningPlatform> GetNotificationPlatform()
     {
-        return wil::CoCreateInstance<NotificationsLongRunningPlatform, INotificationsLongRunningPlatform>(CLSCTX_LOCAL_SERVER);
+        // If supported, allow use of class registrations from less trusted sources.
+        auto supportMarker{ wil::CoCreateInstanceNoThrow<ISupportAllowLowerTrustActivation>(CLSID_ActivationCapabilities) };
+        if (supportMarker)
+        {
+            return wil::CoCreateInstance<INotificationsLongRunningPlatform>(CLSID_NotificationsLongRunningPlatform, CLSCTX_LOCAL_SERVER |
```

Comment:
So Enabling the lower trust flag allows for what exactly here? LRP relaunching the process or callbacks registered with LRP being invoked correctly?

Reply:
It allows elevated clients to get a reference to the LRP platform object.

The lower trust flag allows an elevated process to CoCreateInstance a lower trust COM server, which is in our case the elevated client -> LRP.